modules/commerce_option_pricing/tests/src/Kernel/OptionPriceResolverTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Drupal\Tests\commerce_option_pricing\Kernel;
+
+use Drupal\commerce\Context;
+use Drupal\commerce_option\Entity\ProductOption;
+use Drupal\commerce_option\Entity\ProductOptionValue;
+use Drupal\commerce_price\Price;
+use Drupal\commerce_product\Entity\Product;
+use Drupal\commerce_product\Entity\ProductType;
+use Drupal\commerce_product\Entity\ProductVariation;
+use Drupal\Tests\commerce\Kernel\CommerceKernelTestBase;
+
+/**
+ * Tests the option price resolver.
+ *
+ * @group commerce_option_pricing
+ */
+class OptionPriceResolverTest extends CommerceKernelTestBase {
```

Comment:
I think with the new changes, chances are, these tests might not be passing. If not, we can remove it from this MR and work on fixing it later as we are short on time.

Reply:
I have modified the test to follow the new code. So it should be working now.